Sources/NIOHTTPResponsiveness/HTTPDrippingDownloadHandler.swift

```diff
@@ -0,0 +1,228 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HTTPTypes
+import NIOCore
+import NIOHTTPTypes
+
+/// HTTP request handler sending a configurable stream of zeroes
```

Comment:
Can we document that this requires the use of the HTTPTypes types, not the regular NIO ones.

Reply:
Can I suggest that, for now, we don't make the two "helper" handlers `public`? It's generally wise for us not to commit to too much API without a clear use-case. Making something `public` that was previously `internal` is generally easy, changing something that we accidentally made `public` is much harder.

Reply:
The two "helper" handlers you're referring to are the HTTPDrippingDownloadHandler and the HTTPReceiveDiscardHandler, right? The idea being only the ResponsivenessConfig and SimpleResponsivenessRequestMux end up being public?

I think doing that would end up making this more of an example than a building block. I find that request multiplexers are typically not very reusable (hence the prefix "Simple"). My expectation is that someone wanting to put together a robust implementation would have their own request mux and add things like limits on download size, delay durations, etc.